app/repositories/IndividualCheckRepository.scala

```diff
@@ -77,7 +97,14 @@ class IndividualCheckRepository @Inject()(mongo: ReactiveMongoApi)(implicit ec:
   def setCounter(id: String, attempts: Int): Future[BinaryResult] = {
 
     val selector = Json.obj("id" -> Json.toJson(id))
-    val modifier = Json.obj("$set" -> Json.obj("attempts" -> attempts))
+    val modifier = Json.obj(
+      "$set" -> Json.obj(
+        "attempts" -> attempts,
+            "lastUpdated" -> Json.obj(
+              "$date" -> Timestamp.valueOf(LocalDateTime.now)
```

Comment:
@adamconder Do we need to use `MongoDateTimeFormats` for this?

Reply:
That was taken from trusts-store but we could implement MongoDateTime instead!

Reply:
@johnpoolerperea so now we have the MongoDateTimeFormats I believe this should then become
`"lastUpdated" -> Json.toJson(LocalDateTime.now)(MongoDateTimeFormats.localDateTimeWrite)`

And probably don't need to extend MongoDateTimeFormats on line 35